yt_dlp/extractor/playsuisse.py

```diff
@@ -33,26 +34,60 @@ class PlaySuisseIE(InfoExtractor):
                 'title': 'Der Läufer',
                 'description': 'md5:9f61265c7e6dcc3e046137a792b275fd',
                 'duration': 5280,
-                'episode': 'Der Läufer',
-                'thumbnail': 'md5:44af7d65ee02bbba4576b131868bb783'
+                'thumbnail': 'md5:fa67fb5c1f8fae7321f71b4fbdfda617'
             }
-        },
-        {
-            'url': 'https://www.playsuisse.ch/watch/817193/0',
```

Comment:
Why is this removed?

Reply:
Just because it was also an episode of a series, same as the first test. So it didn't add much value.